docs/platform/howto/okta-user-provisioning-with-scim.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Configure user provisioning for Okta
+---
+
+You can automate user provisioning with Okta through System for Cross-domain Identity Management (SCIM).
+This means you can manage your users and their profiles in one place, Okta, and push
+those changes to the Aiven platform.
+
+Aiven’s integration with Okta supports these features:
+
+- **Push new users**  - Users created in Okta are automatically created as managed users
+  in Aiven.
+- **Push profile updates** - User profile updates in Okta are pushed to Aiven. Profiles
+  for these users cannot be changed in Aiven.
+- **Push user deactivation** - Users that are deactivated or removed in Okta are
+  deactivated in Aiven. You can manually delete users in Aiven after they are deactivated.
+- **Push groups** - Groups created or updated in Okta are created and updated in Aiven.
+
+## Prerequisites
+
+- [Add a verified domain to your organization](/docs/platform/howto/manage-domains)
+- [Add Okta as an identity provider](/docs/platform/howto/saml/add-okta-idp)
+
+## Configure user provisioning for Okta
+
+1. In Okta, click **Applications** and go to the Aiven application.
+1. Click **Provisioning**.
+1. Click **Settings** > **Integration** > **Configure API Integration**.
+1. Select **Enable API Integration**.
+1. In the **API Token field**, paste the **Access token** from the Aiven Console.
+1. Click **Test API Credentials** to confirm the connection is working
+   and save the configuration.
+1. Click **Sign On**.
+1. In the **Credentials Details** section, select **Email** as the
+   **Application username format**.
+1. Click **Save**.
+
+You can assign Okta users to the Aiven application. They get an invite to your Aiven
```

Comment:
Users provisioned by SCIM will be managed users and will be members of organization without going through invite process.